web/js/notifications/wv.notifications.ui.js

```diff
@@ -58,6 +58,10 @@ wv.notifications.ui = wv.notifications.ui || function(models, config) {
    */
   var init = function() {
     var reactComponent, options, p, alertUser;
+    if(!wv.util.browser.localStorage) {
+      console.warn('User does not have localStorage');
```

Comment:
`console.warn` can cause some older browsers to crash, we might want to leave it out, and just return in one line;
```
if(!wv.util.browser.localStorage) return;
```